projects/1.20/assets/meet-your-fight/meetyourfight/lang/zh_cn.json

```
@@ -0,0 +1,149 @@
+{
+	"item.meetyourfight.summon.use": "召唤时会被消耗",
+	"item.meetyourfight.summon.gear": "推荐装备：%s",
+
+	"item.meetyourfight.haunted_bell": "缠魂铃铛",
+	"item.meetyourfight.haunted_bell.desc": "召唤敲钟人",
+	"item.meetyourfight.haunted_bell.gear": "铁质或钻石质",
+	"item.meetyourfight.phantoplasm": "幽魂质",
+	"item.meetyourfight.passages_toll": "旅人之钟",
+	"item.meetyourfight.passages_toll.desc": "将你传送至最多%1$d格远的任意方块上",
+	"item.meetyourfight.spectres_eye": "幽灵之眼",
+	"item.meetyourfight.spectres_eye.desc": "使半径%1$d格内的敌对生物发光",
+	"item.meetyourfight.spectres_grasp": "幽灵之握",
+	"item.meetyourfight.phantasmal_rifle": "幻影步枪",
+	"item.meetyourfight.phantasmal_rifle.desc": "弹射物能够穿过方块",
+	"item.meetyourfight.aether_glazed_cupcake": "浇汁以太蛋糕",
+
+	"item.meetyourfight.devils_ante": "恶魔赌注",
+	"item.meetyourfight.devils_ante.desc": "召唤福尔图娜女爵",
+	"item.meetyourfight.devils_ante.gear": "钻石质",
+	"item.meetyourfight.fortunes_favor": "命运之眷",
+	"item.meetyourfight.slicers_dice": "切割者之骰",
+	"item.meetyourfight.slicers_dice.desc": "攻击时有%1$d%%概率额外造成%2$d%%的伤害",
+	"item.meetyourfight.ace_of_iron": "铁之王牌",
+	"item.meetyourfight.ace_of_iron.desc": "有%1$d%%概率无视受到的攻击",
+	"item.meetyourfight.cocktail_cutlass": "甜心鸡尾酒之刃",
+	"item.meetyourfight.cocktail_cutlass.desc": "攻击时有%1$d%%概率给予目标药水效果",
+	"item.meetyourfight.cocktail_shotgun": "猎人霰弹枪",
+	"item.meetyourfight.cocktail_shotgun.desc": "有%1$d%%概率射出带负面效果的喷溅型药水",
+	"item.meetyourfight.velvet_fortune": "幸运天鹅绒",
+
+	"item.meetyourfight.desc.luck": "会被幸运影响",
+
+	"item.meetyourfight.fossil_bait": "化石之饵",
+	"item.meetyourfight.fossil_bait.desc": "召唤沼泽巨颌怪",
+	"item.meetyourfight.fossil_bait.gear": "石质或铁质",
+	"item.meetyourfight.mossy_tooth": "藓牙",
+	"item.meetyourfight.bone_raker": "骨质钉耙",
+	"item.meetyourfight.depth_star": "深境之星",
+	"item.meetyourfight.depth_star.desc": "可以释放短程冲击波",
+	"item.meetyourfight.depth_star.desc2": "其伤害随普通攻击伤害的增加而增加",
+	"item.meetyourfight.caged_heart": "囚笼之心",

```

这里可以不需要介词，因为是 adj. + n.
```suggestion
	"item.meetyourfight.caged_heart": "笼中心",
```